dChatServer/PlayerContainer.h

```diff
@@ -42,6 +42,8 @@ struct PlayerData {
 	std::string playerName;
 	std::vector<FriendData> friends;
 	std::vector<IgnoreData> ignoredPlayers;
+	uint8_t GMLevel = 0;
```

Comment:
Why is eGameMasterLevel not used here?  In any case this is unused in this PR and should be documenated as a TODO to implement

Reply:
noted, was like 3am and i forgot we had an emun for it lol